docs/src/main/paradox/docs/fusion/architecture.md

```diff
@@ -9,22 +9,23 @@ into native browser JavaScript, during a build step, before being served.
 The build step produces a server artifact to run on a Node.js server, with client-side assets and Javascript.
 
 We produce a docker image that is able to serve the compiled assets directly, which is available on
-@link:[Dockerhub](https://hub.docker.com/repository/docker/bluebrain/nexus-web){ open=new }.
+@link:[Dockerhub](https://hub.docker.com/repository/docker/bluebrain/nexus-web){ open=new }.ode.js server, the client communicates directly to
 
-Although the application is served by a Node.js server, the client communicates directly to
 @ref:[Nexus Delta](../delta/index.md) using @ref:[Nexus.js](../utilities/index.md#nexus-js).
 
-## SubApps
+## Pages
 
-Nexus Fusion is divided into sections called SubApps. SubApps are separate workspaces that attempt to contain the
-concerns of disparate users and activities, and provide different access rights for each.
+Nexus Fusion has undergone a significant restructuring to transition from a SubApp-based architecture to a page-based structure. This change will prepare the application for a full migration to file-system based routing technologie and help seperating the application main features.
```

Comment:
even before the links were cleanly maps to the main feature, it's seems okey to put the real name of the method we used but not get into deep, and in the same time file-system is clear enough that the routes will take the name of the file (even for now it's not because we already say that it's a preparation to migration)